src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -478,6 +478,7 @@ internal static int GetWarningLevel(ErrorCode code)
                 case ErrorCode.WRN_SwitchExpressionNotExhaustiveWithWhen:
                 case ErrorCode.WRN_SwitchExpressionNotExhaustiveForNullWithWhen:
                 case ErrorCode.WRN_RecordNamedDisallowed:
+                case ErrorCode.WRN_OnlyOneOfGetHashCodeAndEqualsIsDefined:
```

Comment:
WRN_EqualsWithoutGetHashCode is level 3, this should be the same.

Reply:
@333fred WRN_EqualsWithoutGetHashCode is about `object.Equals`. But for records, it's about `IEquatable<T>.Equals`. So we can't reuse WRN_EqualsWithoutGetHashCode.

Reply:
I would still prefer it to be at the same level as the existing warning though